controllers/app.controllers.js

```diff
@@ -21,8 +21,25 @@ exports.getArticleById = async (req, res, next) => {
 };
 
 exports.getArticles = async (req, res, next) => {
+	const { topic, sort_by, order_by } = req.query;
+
+	if (topic && !['mitch', 'cats'].includes(topic)) {
```

Comment:
The problem with this is that it will require updating as topics are added, and doesn't account for the differences between the test and development datasets. Rather than checking if a topic is valid here, can you think of another way of verifying if the category exists or not?